src/service/referral-api/src/FamilyHubs.Referral.Core/Commands/CreateReferral/CreateReferralCommand.cs

```diff
@@ -118,26 +108,29 @@ private Data.Entities.Referral AttachExistingStatus(Data.Entities.Referral entit
         {
             entity.Status = referralStatus;
         }
+
         return entity;
     }
 
     private Data.Entities.Referral AttachExistingUserAccount(Data.Entities.Referral entity)
     {
         UserAccount? professional = _context.UserAccounts.SingleOrDefault(x => x.Id == entity.UserAccount.Id);
-        if (professional != null) 
+        if (professional != null)
```

Comment:
Could we use is not null instead of != null. The != (and ==) can potentially be overridden and change the behaviour whereas is (is not) null check the objects nullability. Same elsewhere